kratos/utilities/table_stream_utility.h

```diff
@@ -40,7 +40,7 @@ namespace Kratos
 ///@name  Functions
 ///@{
     
-class KRATOS_API(KRATOS_CORE) TableStreamUtility
+class TableStreamUtility
```

Comment:
Thanks, but last time when we had problems with Windows compilation  the solution was precisely to add this line

Reply:
I will try to compile this in windows, but pretty sure that was never working in windows in the first place.